### ISSUE TITLE
Style things

### ISSUE THREAD

**Issue by mostarz93** [credmark/credmark-website-v1#134]
Subtile font and color is weird

Main banner is boring…missing something

**Comment by elektramurphy** [comment]
@mostar93 @StijnDobb Yep happy to update - time wise do we want Stijn to design a new banner for this release? Not sure what everyones workload is atm
